openedx/core/djangoapps/user_authn/views/auto_auth.py

```diff
@@ -32,6 +33,7 @@
 )
 from student.helpers import authenticate_new_user, do_create_account
 from util.json_request import JsonResponse
+from util.password_policy_validators import normalize_password
```

Comment:
Not sure why these two changes are present. Doesn't look like either are used here presently, but maybe they should be below in the auto_auth method. If not, remove the imports